pontoon/base/templates/locale_selector.html

```diff
@@ -28,8 +28,8 @@
     {% endif %}
     <ul>
       {% for l in locales %}
-      <li class="clearfix{% if locale and locale == l %} current{% endif %}{% if l and l.chart %} filter{% endif %}">
-        <span class="language {{ l.code|lower }}">{% if l and l.chart %}<a href="{% if project %}{{ url('pontoon.locale.project', l.code, project.slug) }}{% else %}{{ url('pontoon.locale', l.code) }}{% endif %}" class="clearfix">{% endif %}{{ l.name }}{% if l and l.chart %}</a>{% endif %}</span>
+      <li class="clearfix{% if l and l.chart %} filter{% endif %}">
+        <span class="language {{ l.code|lower }}" {% if locale %}{% for key, value in l.serialize().iteritems() %}data-{{ key }}="{{ value }}"{% endfor %}{% endif %}>{% if l and l.chart %}<a href="{% if project %}{{ url('pontoon.locale.project', l.code, project.slug) }}{% else %}{{ url('pontoon.locale', l.code) }}{% endif %}" class="clearfix">{% endif %}{{ l.name }}{% if l and l.chart %}</a>{% endif %}</span>
```

Comment:
It'd be cleaner to just embed the dict as a JSON object in a data attribute. Avoids issue with keys being invalid in HTML and such.